lib/commands/general.js

```diff
@@ -236,10 +236,14 @@ commands.setUrl = async function (url) {
 };
 
 commands.mobileDeepLink = async function (opts = {}) {
-  const {url, package: pkg} = opts;
-  return await this.adb.startUri(url, pkg);
+  const {
+    url,
+    package: pkg
```

Comment:
Please destruct waitForAppLaunch  explicitly from opts and add a doctring to this method

Reply:
Is it something like below?
commands.mobileDeepLink = async function (opts = {}) {
  const {
    url,
    package: pkg,
    waitForLaunch: wait,
  } = opts;
  return await this.adb.startUri(url, pkg, wait);
};

Reply:
something like

```
const {
  url,
  package,
  waitForLaunch,
} = opts;
return await this.adb.startUri(url, package, {waitForLaunch});
```

Reply:
Yes this also worked for me. Let me do it.
And docstring was not there you want me to add that too?

Reply:
yes, I know. But we want to make it better, don't we?

Reply:
Very true @mykola-mokhnach.